tests/framework/data/SqlDataProviderTest.php

```diff
@@ -32,7 +32,7 @@ public function testTotalCount()
             'sql' => 'select * from `customer`',
             'db' => $this->getConnection(),
         ]);
-        $this->assertSame(3, $dataProvider->getTotalCount());
+        $this->assertSame(3, (int) $dataProvider->getTotalCount());
```

Comment:
@klimov-paul Are you want to say that it is logical that `getTotalCount()` or `count()` should resturn string instead of integer?